src/mca/schizo/schizo.h

```diff
@@ -131,6 +133,7 @@ typedef struct {
     prrte_schizo_base_module_init_fn_t                   init;
     prrte_schizo_base_module_define_cli_fn_t             define_cli;
     prrte_schizo_base_module_parse_cli_fn_t              parse_cli;
+    prrte_schizo_base_module_parse_deprecated_fn_t       parse_deprecated;
```

Comment:
Can we change this to `parse_deprecated_cli` since it is just for the CLI deprecated options?

Reply:
I guess I don't really care

Reply:
It will make it cleaner if we have to add a `parse_deprecated_env` option (or something like that) in the future.